api/routes/timeseries.go

```diff
@@ -51,6 +53,19 @@ func TimeseriesHandler(metaCtor api.MetadataStorageCtor, ctorStorage api.DataSto
 			handleError(w, errors.Wrap(err, "Unable to parse post parameters"))
 			return
 		}
+		t, ok := params["timeseriesUris"].([]interface{})
+		if !ok {
+			handleError(w, errors.New("Missing timeseriesUris from query"))
+			return
+		}
+		timeseriesUris := []string{}
```

Comment:
Acronyms should maintain case - https://github.com/golang/go/wiki/CodeReviewComments#initialisms

I don't think we've done this universally, but we should try to adhere to the standard.  The `params` look up string is tied to the client side JSON, so nothing you can do there.  I think the mixed case for URI has slipped in in a few spots beyond this.